lib/windows/app/actions/deviceActions.js

```diff
@@ -160,36 +162,43 @@ const NORDIC_VENDOR_ID = 0x1915;
 const NORDIC_BOOTLOADER_PRODUCT_ID = 0x521f;
 
 function logDeviceListerError(error) {
-    if (error.usb) {
-        // On win32 platforms, a USB device with no interfaces bound
-        // to a libusb-compatible-driver will fail to enumerate and trigger a
-        // LIBUSB_* error. This is the case of a nRF52840 in DFU mode.
-        // We don't want to show an error to the user in this particular case.
-        if (error.message === 'LIBUSB_ERROR_NOT_FOUND' && error.usb.deviceDescriptor) {
-            const { idProduct, idVendor } = error.usb.deviceDescriptor;
-            if (idVendor === NORDIC_VENDOR_ID && idProduct === NORDIC_BOOTLOADER_PRODUCT_ID) {
-                return;
+    return dispatch => {
+        if (error.usb) {
+            // On win32 platforms, a USB device with no interfaces bound
+            // to a libusb-compatible-driver will fail to enumerate and trigger a
+            // LIBUSB_* error. This is the case of a nRF52840 in DFU mode.
+            // We don't want to show an error to the user in this particular case.
+            if (error.message === 'LIBUSB_ERROR_NOT_FOUND' && error.usb.deviceDescriptor) {
+                const { idProduct, idVendor } = error.usb.deviceDescriptor;
+                if (idVendor === NORDIC_VENDOR_ID && idProduct === NORDIC_BOOTLOADER_PRODUCT_ID) {
+                    return;
+                }
             }
-        }
 
-        const usbAddr = `${error.usb.busNumber}.${error.usb.deviceAddress}`;
+            const usbAddr = `${error.usb.busNumber}.${error.usb.deviceAddress}`;
 
-        let message = `Error while probing usb device at bus.address ${usbAddr}: ${error.message}. `;
-        if (process.platform === 'linux') {
-            message += 'Please check your udev rules concerning permissions for USB devices, see ' +
-                'https://github.com/NordicSemiconductor/nrf-udev';
-        } else if (process.platform === 'win32') {
-            message += 'Please check that a libusb-compatible kernel driver is bound to this device, see https://github.com/NordicSemiconductor/pc-nrfconnect-core/blob/master/doc/win32-usb-troubleshoot.md';
-        }
+            let message = `Error while probing usb device at bus.address ${usbAddr}: ${error.message}. `;
+            if (process.platform === 'linux') {
+                message += 'Please check your udev rules concerning permissions for USB devices, see ' +
+                    'https://github.com/NordicSemiconductor/nrf-udev';
+            } else if (process.platform === 'win32') {
+                message += 'Please check that a libusb-compatible kernel driver is bound to this device, see https://github.com/NordicSemiconductor/pc-nrfconnect-core/blob/master/doc/win32-usb-troubleshoot.md';
+                message += 'Would like to reload the app to resolve the error?';
```

Comment:
This extra sentence is not needed, makes the message too long and not relevant either, I suggest to remove it.

Reply:
Would it be good if have a question here. I agree that the message is too long here. Maybe separate log message and confirmation message? What do you think?